irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -74,12 +74,16 @@ namespace {
     sql << queries.second;
   }
 
+  /// returns string of concatenates query arguments to pass them further, if
+  /// error happened
+  using QueryArgsLambda = std::function<std::string()>;
+
   iroha::expected::Error<iroha::ametsuchi::CommandError> makeCommandError(
       std::string &&command_name,
       const iroha::ametsuchi::CommandError::ErrorCodeType code,
-      const std::string &error_extra = "") noexcept {
+      QueryArgsLambda &&query_args) noexcept {
```

Comment:
Is there a point in passing an rvalue?

Reply:
At least it's correct from semantic point of view - it's not going to be used, so it's getting moved